pkg/controllers/machinesync/machine_sync_controller.go

```diff
@@ -407,6 +524,44 @@ func (r *MachineSyncReconciler) createOrUpdateCAPIMachine(ctx context.Context, m
 	return ctrl.Result{}, nil
 }
 
+// createOrUpdateMAPIMachine creates a MAPI machine from a CAPI one, or updates
+// if it exists and it is out of date.
+func (r *MachineSyncReconciler) createOrUpdateMAPIMachine(ctx context.Context, mapiMachine *machinev1beta1.Machine, newMAPIMachine *machinev1beta1.Machine) (ctrl.Result, error) { //nolint:unparam
```

Comment:
Let's change all the MAPI and CAPI contractions in the strings of this function to `Cluster API` and `Machine API`